packages/contracts-bedrock/contracts/libraries/SafeCall.sol

```diff
@@ -35,6 +35,19 @@ library SafeCall {
         return _success;
     }
 
+    /**
+     * @notice Helper function that returns `true` if there is not enough gas remaining to cover
+     *         the minimum gas requirement for a call + an optional reserved gas amount.
+     * @param _minGas      The minimum amount of gas that may be passed to the target context.
+     * @param _reservedGas Optional amount of gas to reserve for the caller after the execution
+     *                     of the target context.
+     */
+    function checkMinGas(uint256 _minGas, uint256 _reservedGas) internal view returns (bool _halt) {
```

Comment:
Nit:
> returns `true` if there is not enough gas remaining

This is unintuitive to me. For example, if I just encountered this function in the wild (with a less descriptive name for the return value), ie:

```
bool ret = checkMinGas(_minGas, 0)
```
I would assume that `ret == true` means we _do_ have enough gas.

Reply:
The parameter is named `_halt` for this reason - glad to add some extra documentation here or flip it around.

Reply:
How about: 
- Flip it
- rename to `hasMinGas()`